### ISSUE TITLE
Fail to upload blob when using io.js

### ISSUE THREAD

**Issue by ucin** [Azure/azure-storage-node#78]
Using `blobuploaddownloadsample.js` code, I always get `The MD5 value specified in the request did not match with the MD5 value calculated by the server.`  error when uploading blob.

```
Starting blobuploaddownloadsample.
Created the container media001
Entering uploadBlobs.
{ [Error: The MD5 value specified in the request did not match with the MD5 value calculated by the server.
RequestId:83f0668c-0001-0043-4e2e-db9a27000000
Time:2015-08-20T10:00:05.4120374Z]
  code: 'Md5Mismatch',
  userspecifiedmd5: '2MLq/ZDCZuGaudysxHn4rw==',
  servercalculatedmd5: 'toArex8eBSX6wlXEoWViTw==',
  statusCode: 400,
  requestId: '83f0668c-0001-0043-4e2e-db9a27000000' }
{ [Error: The MD5 value specified in the request did not match with the MD5 value calculated by the server.
RequestId:260faa5e-0001-0026-172e-db2b7a000000
Time:2015-08-20T10:00:10.1948410Z]
  code: 'Md5Mismatch',
  userspecifiedmd5: 'ovSNfpyWBEVGyKcKMu5Frw==',
  servercalculatedmd5: 'dXyRu2Ikaa5iSxBTyzL3Sg==',
  statusCode: 400,
  requestId: '260faa5e-0001-0026-172e-db2b7a000000' }
```

Change to node v0.1.2.5, the file successfully uploaded.

My environment
* Windows 7 x64
* io.js iojs-v3.0.0-x64
* node.js node-v0.12.5-x64

**Comment by sebastian-sch** [comment]
I have the same problem. 

My environment:
- OS X 10.10
- node-v0.12.7 is working 
i- ojs-v3.3.0 and node-v4.0.0 is not working.

**Comment by yaxia** [comment]
Thanks for reporting the issue. We are investigating.

**Comment by yaxia** [comment]
The issue has been fixed in the 0.6.0. Thanks.
